src/pudl/scripts/dbt_helper.py

```diff
@@ -353,11 +354,11 @@ class TableUpdateArgs:
     help="Overwrite existing yaml and row counts. If false command will fail if yaml or row counts already exist.",
 )
 @click.option(
-    "--etl-fast",
-    default=False,
-    is_flag=True,
-    type=bool,
-    help="Update row counts for fast ETL counts.",
+    "--target",
+    default="etl-full",
+    type=click.Choice(["etl-full", "etl-fast"]),
+    show_default=True,
+    help="What dbt target should be used as the source of new row counts.",
```

Comment:
Rather than a flag that's used to specify the non-default option, make it an argument with a limited set of acceptable values, making the target explicit for easier parsing elsewhere (e.g. in the tests) This argument also mirrors the `--target` option in dbt.

Reply:
Agreed, much more practical, and easier to change should we ever have more targets